openquake/commonlib/nrml.py

```diff
@@ -159,6 +159,9 @@ class ValidNode(LiteralNode):
         nodalPlane=valid.nodal_plane,
         occurRates=valid.positivefloats,
         probs_occur=valid.pmf,
+        weight=valid.probability,
+        alongStrike=valid.probability,
+        downDip=valid.probability,
```

Comment:
Look at the function `def slip_list(nodes)`, the validation is there (there is a slip_range from 0 to 360 degrees).